docs/topics/db/fixtures.txt

```diff
@@ -0,0 +1,154 @@
+========
+Fixtures
+========
```

Comment:
I'd add `fixtures-explanation` and refer this instead of `` :doc:`fixture </topics/db/fixtures>` ``
```suggestion
.. _fixtures-explanation:

========
Fixtures
========
```